internal/translations/locales/en-US/out.gotext.json

```diff
@@ -0,0 +1,277 @@
+{
+    "language": "en-US",
```

Comment:
>"language": "en-US", [](http://example.com/codeflow?start=4&length=20)

Is this statement correct:

Eventually,  there will also be a .lci file  here. We will have a pipeline that directs the Microsoft localization tools to generate the translations in the SQL server language sets from this english json file, and that lci file will inform the loc tools to only localize the "language" and "translation" fields in the output json.

We will submit the output of that pipeline to the main branch in the appropriate internal/translations/locales folders.

That lci file would also have some rules to help the translators avoid inappropriate argument reordering and to identify some non-translatable terms in some strings.

Reply:
This is correct. 
In short, we  provide out.gotext.json file as input 
1. Run LsBuild command which takes this json, lci (file created using LCXAdmin to specify what needs to be localized) and lss (parser related settings file, optional) as input and generates .lcg file.
2. The lcg file is what is provided to translators, and they save it as .lcl after translations are added.
3. Run LsBuild command per language which takes lcl file as input and generates localized '.json' file for that language at specified directory.